dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -140,13 +142,14 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     @SuppressWarnings("unchecked")
     public <T extends WithId> ListResult<T> fetchAll(Class<T> model, Function<ListResult<T>, ListResult<T>>... operators) {
-        Kind kind = Kind.from(model);
```

Comment:
See commit message: 'Caching logic was not exactly right for when the cache did not hold the entire dataset (like when we restart and the dataset is in the DB but not in the cache).'

after restart fetchAll was returning back an empty list, since we were trying to only fetch from the cache.